docs-v2/integrations/all/wildix.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Wildix PBX
+sidebarTitle: Wildix PBX
+---
+
+API configuration: [`wildix`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Feature                                                                          | Status                          |
+| -------------------------------------------------------------------------------- | ------------------------------- |
+| [Auth (OAuth)](/guides/oauth)                                                      | ✅                              |
+| [Syncs](/guides/sync) & [Actions](/guides/actions)                                                              | ✅                              |
+| [Nango Proxy](/guides/proxy)                          | ✅   |
+| Auto-pagination                     | 🚫 (time to contribute: &lt;1h) |
+| API-specific rate limits | 🚫 (time to contribute: &lt;1h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+## Getting started
+
+-   [How to register an Application](https://api.accelo.com/docs/#registering-your-application)
```

Comment:
These links are still from Accelo

Reply:
Thanks, fixed

Reply:
Weirdly I still see some accelo links in the latest code!